src/main/proto/mfdz-realtime-extensions.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto2";
+
+import "gtfs-realtime.proto";
+
+option java_package = "de.mfdz";
+package transit_realtime;
+
+message TripDescriptorExtension {
+  optional string route_url = 1;
+  optional string agency_id = 2;
+  optional string route_long_name = 3;
+  optional uint32 route_type = 4;
+}
+
+extend TripDescriptor {
+  // https://groups.google.com/d/msgid/gtfs-realtime/255580b2-1fce-4605-a854-3e18f24fcab7n%40googlegroups.com
+  optional TripDescriptorExtension trip_descriptor = 1010;
+}
+
+message StopTimePropertiesExtension {
+  enum DropOffPickupType {
+    // Regularly scheduled pickup/dropoff.
+    REGULAR = 0;
+
+    // No pickup/dropoff available
+    NONE = 1;
+
+    // Must phone agency to arrange pickup/dropoff.
+    PHONE_AGENCY = 2;
+
+    // Must coordinate with driver to arrange pickup/dropoff.
+    COORDINATE_WITH_DRIVER = 3;
```

Comment:
In the GTFS-RT I'm trying to stay as close to the GTFS spec as possible and there the text says the following:

> 0 or empty - Regularly scheduled pickup.
> 1 - No pickup available.
> 2 - Must phone agency to arrange pickup.
> 3 - Must coordinate with driver to arrange pickup.

Isn't that what the mapper is there for: to map from the GTFS types to our internal model?

Reply:
Do you feel that the GTFS types and the internal ones need to be named the same?

Reply:
I feel that it lower the amount of keywords you have to remember :-) but not necessary, see it as a potential minor suggestion. If complicated to change the gtfs-extension proposal (as it's submitted to gtfs project I guess?) then fine to leave it like that for me.

Reply:
We would like to get it into the spec, yes. Whether phone/call will make a difference I don't know. If it's ok with you I would like to leave it as it is.

Reply:
FYI: In general we should use the term/wording in the GTFS spec if possible (it uses phone), if not there should be a comment explaining the term (and origin if appropriate). If you change it in this PR - that would be nice, but it is not required to do so.

Reply:
I went with the GTFS terminology rather than what the internal model uses. In GTFS it is called "phone agency" not "call agency": https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#stop_timestxt